### PR TITLE
Add SPI Playground app launch analytics

### LINKED ISSUE #1130: Add Plausible event for opening the SPI Playgrounds app
Attach to any `spi-playgrounds:` link as it's impossible to see what usage the app is truly getting.

### PR DESCRIPTION
**Pull request by daveverwer**
Fixes #1130

### CODE REVIEW

File: Resources/Scripts/playgrounds_app_link.js [hunk 1]
```diff
@@ -29,6 +31,11 @@ export class SPIPlaygroundsAppLinkFallback {
         setTimeout(() => {
           explainerElement.classList.add('hidden')
         }, 5000)
+
+        // Trigger an analytics event and pass the dependency as a custom property
+        const targetUrl = new URL(linkElement.href)
+        const dependencies = targetUrl.searchParams.get('dependencies')
+        measurePlausibleEvent('SPI Playgrounds Launch', { props: { dependency: dependencies } })
```
Comment (daveverwer) [inline Resources/Scripts/playgrounds_app_link.js:38]: Deliberately using the singular `dependency` here so we can measure this in a different way if/when we support multiple dependencies.

Comment (finestructure) [reply Resources/Scripts/playgrounds_app_link.js:38]: So is this reporting on any url with `dependencies=` as a parameter? Should we also look for `spi-playgrounds://open` in the url? Or is this configured server side somewhere?

**Review by finestructure — APPROVED** [review]

**Comment by daveverwer** [comment]
Tagged as `2.35.1`

File: Resources/Scripts/playgrounds_app_link.js (hunk 1 above)
Comment (daveverwer) [reply Resources/Scripts/playgrounds_app_link.js:38]: This code only executes on those links, that check is further up ^^

Comment (finestructure) [reply Resources/Scripts/playgrounds_app_link.js:38]: Oh, I see, it was already in and not part of the diff :+1:

**Pull request merged**